src/main/scala/org/apache/spark/ml/feature/RapidsPCA.scala

```diff
@@ -16,8 +16,8 @@
 
 package org.apache.spark.ml.feature
```

Comment:
Not part of this PR, but ideally this should not be directly in the Apache Spark package.  We should try as much as possible for this to be in a `com.nvidia.spark....` package.  If something needs to stay under the Apache Spark package then we should minimize its footprint as much as possible and put it under a new package within the Apache Spark package (e.g.: org.apache.spark.ml.feature.rapids)

Reply:
Noted.